zenodo/modules/records/data/objecttypes.json

```diff
@@ -479,6 +480,26 @@
   },
   "csl": "article"
 },
+{
+  "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+  "internal_id": "publication-part",
+  "id": "https://zenodo.org/objecttypes/publication/part#",
+  "title": {
+    "en": "Part"
+  },
+  "title_plural": {
+    "en": "Parts"
+  },
+  "schema.org": "https://schema.org/ImageObject",
```

Comment:
``schema.org``, ``datacite``, ``eurepo``, ``openaire`` and ``csl`` attributes have to be updated as well.